dns.tf

```diff
@@ -42,6 +42,21 @@ resource "aws_route53_record" "thanos_loadbalancer" {
   provider = aws.management_dns
 }
 
+resource "aws_route53_record" "grafana_loadbalancer" {
+  count   = local.is_management_env ? 1 : 0
+  name    = "grafana.${local.fqdn}"
+  type    = "A"
+  zone_id = data.terraform_remote_state.management_dns.outputs.dataworks_zone.id
+
+  alias {
+    evaluate_target_health = false
+    name                   = aws_lb.monitoring[0].dns_name
+    zone_id                = aws_lb.monitoring[0].zone_id
+  }
+
+  provider = aws.management_dns
```

Comment:
Can you put the provider as the first line in the resource? Just clearer and more consistent

Reply:
resolved